install.sh

```diff
@@ -42,7 +42,11 @@ for repo in home-env \
     if [[ -d $repo ]]; then
         (
             cd "$repo"
-            git pull -q
+            if [[ ${bivio_home_env_ignore_git_dir_ownership:+1} ]]; then
+                git -c 'safe.directory=*'  pull  -q
```

Comment:
I think this could just be:

`git ${bivio_home_env_ignore_git_dir_ownership:+-c 'safe.directory=*'} pull -q`

Reply:
ba6b7d9